README.md

```diff
@@ -239,10 +239,17 @@ byte[] secretA = encapsulationResult.getSecret();
 EncryptedMessage encryptedMessage = encapsulationResult.getEncryptedMessage();
 ```
 
-The encrypted message `encryptedMessage` is transported back to `BOB` who uses the public key and cipher text included in the message for the decapsulation phase of KEM:
+The encrypted message `encryptedMessage` is converted into a byte array representation `encodedMessage` and transported back to `BOB` 
+who uses the public key and cipher text included in the message for the decapsulation phase of KEM.
+Obtain the byte array from the encrypted message for transport over network:
+```java
```

Comment:
Use a newline before the code samples to assure correctly generated code on the Developer Portal.